Tasks/AzureAppServiceManage/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -164,5 +164,6 @@
   "loc.messages.WebJobAlreadyInStoppedState": "WebJob '%s' is already in stopped state.",
   "loc.messages.RestartingKuduService": "Restarting Kudu Service.",
   "loc.messages.RestartedKuduService": "Kudu Service restarted.",
-  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s."
+  "loc.messages.FailedToRestartKuduService": "Failed to restart kudu Service. %s.",
+  "loc.messages.CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for Managed Service Principal. Please configure Managed Service Identity (MSI) for virtual machine 'https://aka.ms/azure-msi-docs'. Status code: '%s', status message: %s"
```

Comment:
Would this happen if the IMDS service is not configured? or though it is configured, we could end up with some issues hence the message?

Reply:
Changed message for both of them separately .